cpp/cmake/thirdparty/CUDF_GetCPM.cmake

```diff
@@ -17,3 +17,12 @@ if(NOT (EXISTS ${CPM_DOWNLOAD_LOCATION}))
 endif()
 
 include(${CPM_DOWNLOAD_LOCATION})
+
+function(fix_cmake_global_defaults target)
```

Comment:
Can we add a comment as to what this function does?

Reply:
We may want to also consider moving this to a different file

Reply:
I could just use the one in `cudf/cpp/cmake/thirdparty` instead. What do you think about that? I usually dislike using parent directories like that but in this case it might make sense since it is a verbatim copy of the file there.

Reply:
@robertmaynard suggested putting it here because we'll need it everywhere we do `CPMFindPackage`

Reply:
Okay, can we just add a comment?